pkg/reconciler/configuration/resources/revision.go

```diff
@@ -40,8 +40,17 @@ func MakeRevision(config *v1alpha1.Configuration) *v1alpha1.Revision {
 
 	UpdateRevisionLabels(rev, config)
 
-	if rev.Annotations == nil {
-		rev.Annotations = make(map[string]string)
+	// Populate the CreatorAnnotation from configuration.
+	cans := config.GetAnnotations()
+	creator, ok := cans[serving.CreatorAnnotation]
```

Comment:
This should be `serving.UpdaterAnnotation` for this line. The user that _updated_ the Configuration is the one that _created_ this Revision.

```suggestion
	creator, ok := cans[serving.UpdaterAnnotation]
```